### ISSUE TITLE
Tutorial completion fails on running test 

### ISSUE THREAD

**Issue by danoctavian** [ethereum-optimism/optimism-tutorial#8]
Forked the tutorial here and followed the steps according to readme.
https://github.com/danoctavian/optimism-tutorial/tree/13391946ef39cbed13897b9193980ba0249ecb8b

OS: macos: 10.15.7
yarn version: 1.22.10
node version: v10.23.0
npm version: 6.14.8

```yarn test``` fails with:

```
  Optmistic ERC20
    the basics
      1) "before each" hook for "should have a name"


  8 passing (1s)
  1 failing

  1) Optmistic ERC20
       "before each" hook for "should have a name":
     Error: Transaction reverted without a reason
      at <UnrecognizedContract>.constructor (unknown)
      at process._tickCallback (internal/process/next_tick.js:68:7)
      ```

**Comment by platocrat** [comment]
For reference, @danoctavian's repo uses `@eth-optimism/plugins` v0.0.17, but I can also reproduce this same odd error in both `0.0.17` and `0.0.16`:

<img width="891" alt="Screen Shot 2021-03-06 at 11 23 39 AM" src="https://user-images.githubusercontent.com/37757724/110215268-67541c80-7e6e-11eb-9dee-4c8a9d89020c.png">

I also just tested `@eth-optimism/plugins` v0.0.18 on a freshly cloned repo. Running `yarn test` before creating the optimistic test file runs with all EVM tests passing.

Once I add the OVM tests and run `yarn test`, I reproduce Dan's error shown above.

**Comment by kfichter** [comment]
I have a fix for this locally. Will make a new release ASAP.

**Comment by platocrat** [comment]
Should be fixed by #17.
Feel free to reopen this issue if you feel the need
